lib/package/release-note.js

```diff
@@ -0,0 +1,64 @@
+/** @babel */
+
+import { CompositeDisposable, Disposable } from 'atom'
+import path from 'path'
+import fs from 'fs'
+import marked from 'marked'
+import { show } from '../ui/selector'
+import { readCode } from '../misc/paths'
+
+let subs
+const RELEASE_NOTE_DIR = path.join(__dirname, '..', '..', 'release-notes')
+
+export function activate (startupNoteVersion) {
+  subs = new CompositeDisposable()
+  const view = document.createElement('div')
+  const panel = atom.workspace.addModalPanel({ item: view })
+
+  const inner = document.createElement('div')
+  inner.classList.add('julia-client-release-note')
+  inner.setAttribute('tabindex', "1")
+
+  const closeButton = document.createElement('button')
+  closeButton.classList.add('btn', 'icon', 'icon-remove-close', 'release-note-close-button')
+
+  view.appendChild(closeButton)
+  view.appendChild(inner)
+
+  const showNote = (version, versions) => {
+    const p = path.join(RELEASE_NOTE_DIR, version + '.md')
+    const markdown = readCode(p)
+    inner.innerHTML = marked(markdown)
+    panel.show()
+    view.focus()
+  }
+  const close = () => panel.hide()
+  closeButton.onclick = close
+  const panelView = panel.getElement()
+  panelView.style['max-width'] = '75em'
+
+  subs.add(
+    atom.commands.add('atom-workspace', 'julia-client:open-release-note', () => {
+      const versions = fs.readdirSync(RELEASE_NOTE_DIR)
+        .filter(path => path !== 'README.md')
+        .map(path => path.replace(/(.+)\.md/, 'v $1'))
+      show(versions)
+        .then(version => showNote(version.replace(/v\s(.+)/, '$1')))
+        .catch(err => console.log(err))
+    }),
+    atom.commands.add(panelView, 'julia-client:cancel-release-note',  (event) => {
+      close()
+      event.stopPropagation()
+    }),
+    atom.packages.onDidActivateInitialPackages(() => {
+      if (startupNoteVersion) setTimeout(() => showNote(startupNoteVersion), 500)
+    }),
```

Comment:
on my setup, subscribing on `onDidActivateInitialPackages` was necessary for the modal to persist

Reply:
"persist" as in "not close directly after showing it"? Kinda makes sense, I suppose.

Reply:
I tried to mean, the modal doesn't keep to be shown up without `onDidActivateInitialPackage` maybe because otherwise some other modal implicitly show up on startup and hide (already shown) the release note modal.
I don't think this fix is so robust though